### PR TITLE
Add chain_id for transaction

### PR DESCRIPTION
**Pull request by u2**
Add chain_id and value for Transaction proto

### CODE REVIEW

File: libproto/src/router.rs [hunk 1]
```diff
@@ -74,7 +74,6 @@ pub enum MsgType {
     ExecutedResult,
     SnapshotReq,
     SnapshotResp,
-    LocalSync,
     // Generate MSG-PROTOS struct automatically end.
```
Comment (u2) [inline libproto/src/router.rs:77]: Auto generated.

File: libproto/src/protos/blockchain.proto [hunk 2]
```diff
@@ -51,7 +51,9 @@ message Transaction {
     uint64 quota = 3;
     uint64 valid_until_block = 4;
     bytes data = 5;
-    uint32 version = 6;
+    uint64 value = 6;
+    uint32 chain_id = 7;
+    uint32 version = 8;
```
Comment (driftluo) [inline libproto/src/protos/blockchain.proto:56]: Here will break, increase the field without order

**Comment by TheWaWaR** [comment]
If all tests passed in `cita`, then it's OK to me.

**Review by driftluo — APPROVED** [review]

File: libproto/src/protos/blockchain.proto (hunk 2 above)
Comment (u2) [reply libproto/src/protos/blockchain.proto:56]: There is not `version` in the latest release, all these three are new.

**Pull request merged**